src/main/java/net/snowflake/client/core/SfSqlArray.java

```diff
@@ -16,9 +16,16 @@
 @SnowflakeJdbcInternalApi
 public class SfSqlArray implements Array {
 
+  private String text;
   private int baseType;
   private Object elements;
 
+  public SfSqlArray(String text, int baseType, Object elements) {
```

Comment:
1. can we reuse new constructor in other constructors?
2. can we remove other constructors?